app/Http/Controllers/UserAdminController.php

```diff
@@ -26,20 +26,36 @@ class UserAdminController extends Controller
     /**
      * Display a listing of the resource.
      *
-     * @return \Illuminate\Http\Response
+     * @param Request $request
+     * @return \Illuminate\View\View
      */
     public function index(Request $request)
     {
 
         $search = $request->input('query');
+        $filter = $request->input('filter');
+
+        switch ($filter) {
```

Comment:
@JonaMata Do you have any comments on the queries, I was wondering if perhaps there is an easier or more elegant method?